packages/video_player_videohole/lib/video_player.dart

```diff
@@ -117,6 +118,12 @@ class VideoPlayerValue {
   /// If [hasError] is false this is `null`.
   final String? errorDescription;
 
+  /// True if video has finished playing to end.
+  ///
+  /// Reverts to false if video position changes, or video begins playing.
+  /// Does not update if video is looping.
+  final bool isCompleted;
```

Comment:
Good suggestion, Thank you~

Reply:
Oh I am so sorry.  My comment wasn't about naming. I just thought of invoking a new event when the VideoEventType.completed event occurs. This was a bad idea and I didn't think that there was already an isCompleted on VideoPlayerValue.
https://github.com/flutter/packages/blob/main/packages/video_player/video_player/lib/video_player.dart#L61

Sorry for the inconvenience. VideoPlayerValue.isCompleted is more appropriate.

Reply:
Hi, is it possible to expose `VideoEvent`? So that developers can track all the actual events

Reply:
> Hi, is it possible to expose `VideoEvent`? So that developers can track all the actual events

You can get video event from VideoPlayerValue, VideoPlayerValue has more information than VideoEvent.

Reply:
> This was a bad idea and I didn't think that there

Done, I just copy isCompleted from video_player plugin.